tripper/datadoc/keywords.py

```diff
@@ -178,100 +251,680 @@ def add_theme(
                 else:
                     raise TypeError(f"Unknown theme: {name}")
 
-    def parse(
+    def load_yaml(
         self,
         yamlfile: "Union[Path, str]",
         timeout: float = 3,
     ) -> None:
-        """Parse YAML file with keyword definitions."""
-        # pylint: disable=too-many-nested-blocks
+        """Load YAML file with keyword definitions.
+
+        Arguments:
+            yamlfile: Path of URL to a YAML file to load.
+            timeout: Timeout when accessing remote files.
```

Comment:
This goes into the documentation. Make a link to an example.

Reply:
Ok. Added it to issue #462.